types/serialization_test.go

```diff
@@ -48,6 +48,72 @@ func TestLabels(t *testing.T) {
 	}
 }
 
+func BenchmarkFill(b *testing.B) {
```

Comment:
How these compare before/after?

Reply:
Roughly 20% better if memory serves me and my notes are accurate.